UserInterfaces/animalTableViewCell.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  AnimalTableViewCell.swift
+//  Paws4Me
+//
+//  Created by Kirsty-Lee Walker on 2022/02/26.
+//
+
+import Foundation
+import UIKit
+
+class AnimalTableViewCell: UITableViewCell {
+    
+    @IBOutlet weak var imagePetView: UIImageView!
+    @IBOutlet weak var petNameLabel: UILabel!
+    
+    var index = 0
+    var pet: AdoptPet! {
+        didSet {
+            updateUI()
+        }
+    }
+    
+    func updateUI() {
+        guard let url = URL(string: (pet.page?[index].animalImage)!) else { return }
```

Comment:
Force unwrapping here.

Reply:
Great find. I completely forgot about that.